src/SmartComponents/BaselinesTable/redux/__tests__/baselinesTableReducer.fixtures.js

```diff
@@ -1,22 +1,27 @@
 import moment from 'moment';
 
 /*eslint-disable camelcase*/
-const baselinesListPayload = ([
-    {
-        created: '2019-01-18T13:30:00.000000Z',
-        display_name: 'beavs baseline',
-        fact_count: 2,
-        id: '1234',
-        updated: '2019-05-18T15:00:00.000000Z'
-    },
-    {
-        created: '2019-02-18T13:30:00.000000Z',
-        display_name: 'micjohns baseline',
-        fact_count: 2,
-        id: 'abcd',
-        updated: '2019-05-19T15:00:00.000000Z'
+const baselinesListPayload = ({
+    data: [
+        {
+            created: '2019-01-18T13:30:00.000000Z',
+            display_name: 'beavs baseline',
+            fact_count: 2,
+            id: '1234',
+            updated: '2019-05-18T15:00:00.000000Z'
+        },
+        {
+            created: '2019-02-18T13:30:00.000000Z',
+            display_name: 'micjohns baseline',
+            fact_count: 2,
+            id: 'abcd',
+            updated: '2019-05-19T15:00:00.000000Z'
+        }
+    ],
+    meta: {
+      total_available: 2
```

Comment:
can you add 'count: 2' here also? its not used but just to make it match what's being returned by the API